client/src/components/Login.jsx

```diff
@@ -32,11 +32,9 @@ function Login() {
     e.preventDefault();
     try {
       setLoading(true);
-      await login(email, password)
-        .then((userCredential) => {
-          const { user } = userCredential;
-          console.log('user cred', userCredential);
-          // TODO: add userId to context
+      login(email, password)
```

Comment:
how will we be storing the user credentials?

Reply:
They're stored on the firebase authentication db